src/ios/ShareExtension/ShareViewController.m

```diff
@@ -88,7 +88,8 @@ - (void) openURL:(nonnull NSURL *)url {
             NSInvocation *invocation = [NSInvocation invocationWithMethodSignature:methodSignature];
 
             // Arguments
-            NSDictionary<NSString *, id> *options = [NSDictionary dictionary];
+            UISceneOpenExternalURLOptions * options = [UISceneOpenExternalURLOptions alloc];
```

Comment:
The pattern in obj-c is generally `[[Class alloc] init]`. Are you sure a call to `init` isn't missing here? This might let the object fields uninitialized.

Reply:
Good catch. I missed that.